### ISSUE TITLE
Suggestion: Swiftcast added to the Cooldowns

### ISSUE THREAD

**Issue by captawesom3** [0ceal0t/JobBars#60]
Would it be possible to add Swiftcast the Cooldowns so I know if my Co-healer/Rezbot has it available or prepped already to rez someone.

**Comment by Felys1998** [comment]
Agreed, this is basically the one thing that ZeffUI tracks that this doesnt that I really miss.

**Comment by Felys1998** [comment]
https://github.com/0ceal0t/JobBars/pull/63 slapped together a pull request real quick.
